includes/Admin/Admin_Page.php

```diff
@@ -312,8 +312,17 @@ static function ( Check $check ) {
 	 */
 	public function filter_plugin_action_links( $actions, $plugin_file, $plugin_data, $context ) {
 
+		if ( in_array( $context, array( 'mustuse', 'dropins' ), true ) ) {
+			return $actions;
+		}
+
 		$plugin_check_base_name = plugin_basename( WP_PLUGIN_CHECK_MAIN_FILE );
-		if ( in_array( $context, array( 'mustuse', 'dropins' ), true ) || $plugin_check_base_name === $plugin_file ) {
+		if ( $plugin_check_base_name === $plugin_file ) {
+			$actions[] = sprintf(
+				'<a href="%1$s">%2$s</a>',
+				esc_url( admin_url( 'tools.php?page=plugin-check' ) ),
+				esc_html__( 'Check a Plugin', 'plugin-check' )
```

Comment:
```suggestion
				esc_html__( 'Check a plugin', 'plugin-check' )
```

Or maybe `Run Plugin Check`